tide/master/CMakeLists.txt

```diff
@@ -141,6 +141,7 @@ set(TIDEMASTER_INCLUDE_NAME tide/master)
 if(TIDE_ENABLE_REST_INTERFACE)
   list(APPEND TIDEMASTER_PUBLIC_HEADERS
     rest/JsonOptions.h
+    rest/jsonschema.h
```

Comment:
This naming is intentional, the project follows this naming convention: CamelCase for files which define a class, lowercase for files which don't define a class but contain just constants/includes or helper functions in a namespace (log.h, config.h, types.h, helpers.h, serialization/utils.h, etc...)